guide/creating-your-bot/common-questions.md

```diff
@@ -99,4 +99,8 @@ user.send('<content>');
 });
 ```
 
+### What is the difference between a User and a GuildMember?
+
+A lot of users get confused what is the difference between a User and a GuildMember the simple answer is that an User represent a Discord User Global and a GuildMember represent a Discord User on a specific server that means only GuildMembers can have Permissions, Roles and Nicknames as example because all these things are server bound information that could be different on every server that User is in.
```

Comment:
* Needs a period before "the simple answer".
* "a Discord User Global" should be "a global Discord user"
* Uncapitalize "User" in "... represent a Discord User".
* Needs a period before "the means only GuildMembers ...".
* Uncapitalize "Permissions, Roles and Nicknames" since we're not talking about the classes but rather the actual nouns, and add a comma before the "and".
* "... as example" -> "... , for example, ..."
* "server bound" -> "server-bound"
* Uncapitalize "User" in that last sentence".

Reply:
Another thing is that this is placed awkwardly; it's right in-between the `<p class="tip">` tag and text related to `.awaitMessages()`. This should be placed after that.